notebooks/20200721_TMPRSS2_SVM.ipynb

```diff
@@ -0,0 +1,544 @@
+{
```

Comment:
<p>Hmm, so if I understand correctly, the old code (commented out) utilizes the scaled values, and where scaled values are not available, uses the raw unscaled values.  The new code utilizes the scaled values, and just removes any compounds which don't have scaled values from the training set.  Unless I'm mistaken, that would remove like half the training data.  Sure, nafamostat occurs as a top hit after this change, but that shouldn't be our benchmark for success... that's bad science.  Good validation results are the benchmark for a good method, not achieving a particular result which you assume a priori to be true.  You can see your error distribution is worse after this change, an indication that this change is not a good one.  Data needs to be worse than useless to justify removing it.  I would use all the data, and if nafamostat doesn't show up, so be it; I still trust the results more.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/srensi/tmprss2/pull/25/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='14'/>

Reply:
this has been discussed, resolved.  Using only the scaled values is better.